backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/internal/BpkDialogImpl.kt

```diff
@@ -202,6 +218,32 @@ private fun DialogIcon(icon: Dialog.Icon?) {
     }
 }
 
+private fun Modifier.dialogSurface(): Modifier {
+    return this.then(
+        Modifier.padding(
+            top = IconPadding,
+            start = BpkDimension.Spacing.Lg,
+            end = BpkDimension.Spacing.Lg,
+        ).widthIn(max = 400.dp),
+    )
+}
```

Comment:
can be simplified to 
```
private fun Modifier.dialogWidth(): Modifier =
    widthIn(
        max = DialogMaxWidth,
    ).padding(
        start = BpkDimension.Spacing.Lg,
        end = BpkDimension.Spacing.Lg,
    )
```

Reply:
also can we rename it to `dialogWidth`?

Reply:
Done, but I've left in `top = IconPadding,` because I think we agreed this is good to keep it centered.